types/mongoose/index.d.ts

```diff
@@ -1623,6 +1623,7 @@ declare module "mongoose" {
      * @param bool defaults to true
      */
     lean(bool?: boolean): Query<Object>;
```

Comment:
While you're here, see the second to last bullet [here](https://github.com/DefinitelyTyped/DefinitelyTyped#common-mistakes) about using the type `object` instead of `Object`.